provision/aws/aws_check_subnet_groups.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+function check_db_subnet_group_usage() {
+    local db_subnet_group_name=$1
+    local region=$2
+    # Check for any RDS instances associated with the subnet group
+    rds_instances=$(aws rds describe-db-instances --region "$region" --query "DBInstances[?DBSubnetGroup.DBSubnetGroupName=='${db_subnet_group_name}'].DBInstanceIdentifier" --output text)
+    # Check for any Aurora clusters associated with the subnet group
+    rds_clusters=$(aws rds describe-db-clusters --region "$region" --query "DBClusters[?DBSubnetGroup=='${db_subnet_group_name}'].DBClusterIdentifier" --output text)
+    if [[ -z $rds_instances && -z $rds_clusters ]]; then
```

Comment:
I think we can simplify and instead of the if/else, just have `[[ -z $rds_instances && -z $rds_clusters ]]` as the last command, similar to: https://github.com/keycloak/keycloak-benchmark/blob/e9a17961258bf6497d93c0b476b9c16d38c7482f/provision/aws/rds/aurora_delete_global_db.sh#L8-L15

Reply:
ah, let me try that, will save a few lines of bash script.